charts/lighthouse/config/lighthousejobs.lighthouse.jenkins.io.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Running locally I got a validation error on `apiextensions.k8s.io/v1beta`, so hey, let's switch to `v1`.